index.html

```diff
@@ -353,6 +353,166 @@ <h2 class="skill-title">Skills</h2>
           </article>
         </div>
       </section>
+      <section class="contact-me">
+        <h2 class="contact-title">
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h2>
+        <form
+          action="https://formspree.io/f/xwkydvlz"
+          class="form"
+          method="post"
+        >
+          <fieldset>
+            <input
+              name="fullName"
+              type="text"
+              placeholder="Full name"
+              maxlength="30"
+              required
+            />
+          </fieldset>
+          <fieldset>
+            <input
+              name="email"
+              type="email"
+              placeholder="Email address"
+              required
+            />
+          </fieldset>
+          <textarea name="message" maxlength="500" required>
+Enter text here
+            </textarea
```

Comment:
**[REQUIRED CHANGE]**

- Please, kindly remove the text you added "Enter text here", as this is making your form submit when the Textarea input is empty, instead update by giving it a placeholder = "Enter text here".
```
<textarea name="message" id="message" rows="5" maxlength="500" placeholder="Enter text here" required></textarea>
```

Reply:
**[REQUIRED CHANGES]**

*README FILE*

- Please, kindly update the getting started header and add some steps on how users can have your repo running locally on their machine.
- Also, update the authors by adding others that have contributed to the project.
- Kindly, remove the headers that are not relevant to the project to make your readme appear more professional.
- The live demo link should take users to the live demo view of your portfolio, it takes the user to your GitHub account which is not the expected outcome. Please, kindly update the live demo link or remove it, for now, to improve the user experience when checking your readme file and to make your readme appear more professional.

Reply:
Thank you @zieeco for your feedback, I Will modify it.